calplus/v1/object_storage/client.py

```diff
@@ -1,14 +1,166 @@
+import logging
 import calplus.conf
+
 from calplus.base import BaseClient
+from calplus.exceptions import DriverException
 
 CONF = calplus.conf.CONF
+LOG = logging.getLogger(__name__)
 
 
 class Client(BaseClient):
     """Top-level object to access CAL API
     This class must be extended base.Singleton class to make
     sure only one instance of this one is ever created."""
 
-    def __init__(self, provider, cloud_config, *args, **kwargs):
+    def __init__(self, provider, cloud_config):
         BaseClient.__init__(self, CONF.object_storage.driver_path,
                             provider, cloud_config)
+
+    def create_container(self, container, **kwargs):
```

Comment:
the term container was used only in Swift context, how about S3?
maybe we should use the more generic term or write more details in docstring